### ISSUE TITLE
Filter internal databases

### ISSUE THREAD

**Issue by mokraemer** [BenMorel/mysql-replication-check#4]
Some internal databases should always filtered out:

- information_schema
- performance_schema
- mysql

I suggeste changing loadTables to
```
function loadTables(PDO $master) {
    $filtered=['information_schema','performance_schema','mysql'];
    $databases = [];

    $statement = $master->query('SHOW DATABASES');
    while (false !== ($database = $statement->fetchColumn())) {
        if(!in_array($database,$filtered)){
          $databases[] = $database;
        }
    }

    $tables = [];

    foreach ($databases as $database) {
        $statement = $master->query("SHOW FULL TABLES FROM `$database` WHERE Table_Type != 'VIEW'");
        while (false !== $table = $statement->fetchColumn()) {
            $tables[] = $database . '.' . $table;
        }
    }

    return $tables;
}
```

**Comment by mokraemer** [comment]
[bug4.txt](https://github.com/BenMorel/mysql-replication-check/files/7684273/bug4.txt)

**Comment by BenMorel** [comment]
Fixed in ee9955ae196b0e2fb13c061b5e069f1bf582e2f2.
